spec/lib/embed/viewer/was_seed_spec.rb

```diff
@@ -33,12 +34,15 @@
       expect(html).to have_css '.sul-embed-was-thumb-list', visible: false, count: 1
       expect(html).to have_css '.sul-embed-was-thumb-item', visible: false, count: 4
       expect(html).to have_css '.sul-embed-was-thumb-item-div', visible: false, count: 4
+      expect(html).to have_css '.sul-embed-was-thumb-item-div[style="height: 200px; width: 200px;"]', visible: false, count: 4
```

Comment:
I meant it to make it two lines. The first one focuses on the div display itself, then the next one focuses on the style. It will help anyone in troubleshooting the problem later to determine what the source of the problem is.